projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -72,6 +76,25 @@ public function admin_init() {
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_admin_scripts' ) );
 	}
 
+	/**
+	 * Show a notice (warning) if the Jetpack Protect plugin
+	 * is active and has vulnerabilities.
+	 */
+	public function jetpack_protect_detect_vuls_notice() {
+		echo '<div class="notice notice-warning"><p>';
```

Comment:
We could add a check to avoid the admin_notice to be added in the Protect overview page itself.

(we can also do this by remove_action in the `admin_init` method of this class, which runs when the plugin's page is loaded)